lfs/transfer_queue.go

```diff
@@ -338,7 +338,7 @@ func (q *TransferQueue) individualApiRoutine(apiWaiter chan interface{}) {
 	for t := range q.apic {
 		obj, err := t.LegacyCheck()
 		if err != nil {
-			if q.canRetryObject(obj.Oid, err) {
+			if q.canRetryObject(t.Oid(), err) {
```

Comment:
Confirmed that this fixes the issue, and that the new `test-legacy-retries` would catch it if it still was present. Thanks!